redis/src/aio/connection_manager.rs

```diff
@@ -18,6 +18,62 @@ use std::sync::Arc;
 use tokio_retry::strategy::{jitter, ExponentialBackoff};
 use tokio_retry::Retry;
 
+/// Apply a maximum delay. No retry delay will be longer than this `max_delay`.
+#[derive(Clone, Debug, Default)]
+pub struct RetryStrategyInfo {
+    /// The resulting duration is calculated by taking the base to the `n`-th power,
+    /// where `n` denotes the number of past attempts.
+    pub exponent_base: u64,
```

Comment:
the fields of this struct should be private, in order to prevent breaking the user's code if fields are added.